adagrams/game.py

```diff
@@ -1,11 +1,115 @@
+import random
+import string
+
+
 def draw_letters():
-    pass
+    LETTER_POOL = ETTER_POOL = {
```

Comment:
This can be declared outside of the function so that it does not have to be recreated each time the function is called. Also, it looks like there is a typo here. This should be `LETTER_POOL = {`